apps/docs/lib/custom-content/getCustomContent.ts

```diff
@@ -21,9 +21,9 @@ function contentToCamelCase(feature: CustomContent) {
 }
 
 type CustomContentResult<T extends CustomContent[]> = {
-  [key in CustomContentToCamelCase<T[number]>]:
-    | CustomContentTypes[CustomContentToCamelCase<T[number]>]
-    | null
+  [key in CustomContentToCamelCase<T[number]> extends keyof CustomContentTypes
```

Comment:
Adding new custom content fields doesn't need to touch this file: the `CustomContent.types.ts` fix should fix any current issues